tests/tests.toml

```diff
@@ -152,6 +152,10 @@ asr = true
 llvm = true
 llvm_dbg = true
 
+[[test]]
+filename = "../integration_tests/test_const_access.py"
+asr = true
+
```

Comment:
Please remove this as it is already tested in `integration_tests`. We should add reference tests only when utmost necessary and there is no other viable option to test it. For example, in the case of verifying runtime output or to check generation of a semantic error.

Please also remove the associated generated reference files.